config/crd/patches/cainjection_in_servicebindings.yaml

```diff
@@ -2,6 +2,4 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
why do we need to remove these lines?

Reply:
From what I could understand, this interfered with the same annotation configured by `release-manifest.py`.